cloudslang-entities/src/main/java/io/cloudslang/lang/entities/LoopStatement.java

```diff
@@ -37,4 +40,20 @@ protected LoopStatement() {
     public String getExpression() {
         return expression;
     }
+
```

Comment:
I think that the elegance and code size is not enough in this case...
We will loose in the performance area for each time we need to compare 2 objects or toString() them, and I think it will probably happen a lot...